core/kazoo_globals/src/kz_nodes.erl

```diff
@@ -78,11 +83,16 @@
 -define(SIMPLE_ROW_STR, ?HEADER_COL ": ~s~n").
 -define(SIMPLE_ROW_NUM, ?HEADER_COL ": ~B~n").
 
+-define(NODE_GEN_SECRET, kz_binary:rand_hex(16)).
```

Comment:
Perhaps I should re-phrase :)

I would think that something where we can determine the node in question after the fact without additional information would be better i.e md5(node())